### ISSUE TITLE
[Feature Request] Increase accessibility on color picker component

### ISSUE THREAD

**Issue by cmooreDT** [vuetifyjs/vuetify#21834]
### Problem to solve
All buttons and input fields on the color picker component lack descriptive labels. The screen reader only announces "button" or "slider" or "stepper". Users have no idea what their interactions would change. 

Accessibility should not be skimped, even if some users of screen readers are unlikely to use a color picker.

### Proposed solution
Add appropriate aria-labels to all interactive elements in the color picker.
1. Eyedropper icon button -- "Select color with eyedropper"
2. Slider 1 -- "Adjust RGB/HSL value"
3. Slide 2 -- "Adjust alpha value"
4. Input 1 for RGB/RGBA -- "Red value"
5. Input 2 for RGB/RGBA -- "Green value"
6. Input 3 for RGB/RGBA -- "Blue value"
7. Input 4 for RGBA/HSLA -- "Alpha value"
8. Input 1 for HSL/HSLA -- "Hue value"
9. Input 2 for HSL/HSLA -- "Saturation value"
10. Input 3 for HSL/HSLA -- "Lightness value"
11. Input for HEX -- "HEX value"
12. Input for HEXA -- "HEXA value"
13. Color format toggler -- "Change color format"
